app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -126,3 +128,83 @@ define [
         else
           window.open(data[0].url)
       )
+
+    deleteRecording: (e) ->
+      e.preventDefault()
+      if !confirm I18n.t("Are you sure you want to delete this recording?")
+        $(e.currentTarget).parents('.inline-block').find('.al-trigger').focus()
+        return
+      $deleteButton = $(e.currentTarget).parent()
```

Comment:
This is very magical.  When you put focus on some parents we are lost as to what that is.  It would be nice to tell us in a comment since this is not self documenting really.

Reply:
Right. The button is enclosed in a div element, would it help to add a selector? 

```$deleteButton = $(e.currentTarget).parent('.ig-button')```

or even

```$deleteButton = $(e.currentTarget).parent('div.ig-button')```

 Or an explicit comment would be preferred?

Sorry pal, I am just trying to understand what would make things easier for Engineers to review. I know the person reviewing is not always the same, so I am trying to find middle ground while being consistent with the code and styling in a clearer way.

Reply:
@jfederico You are right its hard to strike a balance especially in coffeescript land.  I would much rather have a selector on this ```$deleteButton = $(e.currentTarget).parent('div.ig-button')``` would be my preferred way.

Reply:
Done